### DISCUSSION TITLE
Please allow pinning my favorite repo on my dashboard.

### DISCUSSION THREAD

**Discussion by LinXueyuanStdio** [community/community#66837] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have to type a lot of words, click search button, wait for a long time, and finally find my target repo in a long list. But I just have up to five repos used frequently.

**Comment by andreslopezferro** [comment]
This can be a nice feature! 🤓

**Comment by Sneizzle** [comment]
Hello. I think you can star repositories that you frequently use or want to access quickly.
It places it in your "Starred" list which is quite nice and easily accesible from your frontpage/dashboard.

You can also pin it but it seems like you've reached Pin Cap?

I can only think of third party solutions beside this. Good Luck

**Comment by LamThanhNgan** [comment]
It should be a nice feature 💯
